### PR TITLE
enhance: can choose chash keys from drop down list

### PR DESCRIPTION
**Pull request by lilien1010**
fix: https://github.com/apache/incubator-apisix-dashboard/issues/64

### CODE REVIEW

File: src/views/schema/upstream/edit.vue [hunk 1]
```diff
@@ -155,6 +167,49 @@ export default class extends Vue {
...
+    {
+      value: 'remote_port',
+      label: 'remote_port'
+    },
+    {
+      value: 'hostname',
+      label: 'hostname'
+    },
+    {
+      value: 'arg_id',
```
Comment (membphis) [inline src/views/schema/upstream/edit.vue]: `arg_id` is wrong, here is the Nginx variable index: http://nginx.org/en/docs/http/ngx_http_core_module.html#var_arg_

`arg_name`: argument name in the request line, the value of the name here is not fixed, eg: `arg_id`, `arg_name`, `arg_age`.

**Comment by juzhiyuan** [comment]
Thanks for your contribution, but does this change need API support? maybe API instance is using the older codes.

image

File: src/views/schema/upstream/edit.vue (hunk 1 above)
Comment (lilien1010) [reply src/views/schema/upstream/edit.vue]: I put the arg_id here just reminder  that arg_*** is able to input here

**Comment by lilien1010** [comment]
@juzhiyuan  you need upgrade APISIX to master ,which conatin my lastest code

**Comment by membphis** [comment]
> @juzhiyuan you need upgrade APISIX to master ,which conatin my lastest code

Please wait for more time, I need @moonming's help.

**Comment by juzhiyuan** [comment]
> @juzhiyuan you need upgrade APISIX to master ,which conatin my lastest code

I see, so we need the demo instance to be updated.

**Comment by juzhiyuan** [comment]
> > @juzhiyuan you need upgrade APISIX to master ,which conatin my lastest code
> 
> Please wait for more time, I need @moonming's help.

Sure, take your time :)

File: src/views/schema/upstream/edit.vue (hunk 1 above)
Comment (juzhiyuan) [reply src/views/schema/upstream/edit.vue]: It looks everything here is ok, I will approve soon.

**Pull request merged**